buffer/src/test/java/io/netty/buffer/AdaptiveByteBufAllocatorTest.java

```diff
@@ -89,6 +89,10 @@ void sliceOrDuplicateUnwrapLetNotEscapeRootParent(boolean slice) {
         AdaptiveByteBufAllocator allocator = newAllocator(false);
         ByteBuf buffer = allocator.buffer(8);
         assertInstanceOf(buffer, AdaptivePoolingAllocator.AdaptiveByteBuf.class);
+        // Unwrap if this is wrapped by a leak aware buffer.
+        if (buffer instanceof SimpleLeakAwareByteBuf) {
```

Comment:
should go one line up

Reply:
It's fine where it is as the assert impl does the unwrapping already